### PR TITLE
Refactored pointwise ops and moved memset operations to a different file.

### PR DESCRIPTION
**Pull request by adityaiitb**

### CODE REVIEW

File: pyprof/prof/memory.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .base import OperatorLayerBase
+from .tensor import Tensor
+
+def readMarker(d):
```
Comment (debermudez) [inline pyprof/prof/memory.py:20]: Would it make sense to have this method as part of a base class between OperatorLayerBase and the 3 classes that use it?

File: pyprof/prof/memory.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    marker = eval(d.argMarker[0])
+    return marker['mod'], marker['op'], marker['args']
+
+class OneZero(OperatorLayerBase):
```
Comment (debermudez) [inline pyprof/prof/memory.py:24]: Can you comment this class to give some indication what it is for?

File: pyprof/prof/memory.py [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    def mod(self):
+        return self.mod_
+
+    def bytes(self):
+        return self.input.bytes
+
+    def flops(self):
+        return 0
+
+class Fill(OperatorLayerBase):
```
Comment (debermudez) [inline pyprof/prof/memory.py:58]: Can you add a brief comment?

File: pyprof/prof/memory.py [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+    def mod(self):
+        return self.mod_
+
+    def bytes(self):
+        return self.input.bytes
+
+    def flops(self):
+        return 0
+
+class Full(OperatorLayerBase):
```
Comment (debermudez) [inline pyprof/prof/memory.py:90]: Brief comment on the class please

**Review by debermudez — CHANGES_REQUESTED** [review]

File: pyprof/prof/memory.py (hunk 2 above)
Comment (adityaiitb) [reply pyprof/prof/memory.py:24]: Done.

File: pyprof/prof/memory.py (hunk 3 above)
Comment (adityaiitb) [reply pyprof/prof/memory.py:62]: Done.

File: pyprof/prof/memory.py (hunk 4 above)
Comment (adityaiitb) [reply pyprof/prof/memory.py:98]: Done.

File: pyprof/prof/memory.py (hunk 1 above)
Comment (adityaiitb) [reply pyprof/prof/memory.py:20]: Discussed offline.

**Review by debermudez — APPROVED** [review]

**Pull request closed without merging**